plex-official.json

```
@@ -0,0 +1,54 @@
+{
+    "Plex-official": {
+        "containers": {
+            "plexinc-pms-docker": {
+                "image": "plexinc/pms-docker",
+                "opts": [
+                    [
+                        "--net=host",
+                        ""
+                    ]
+                ],
+                "launch_order": 1,
+                "ports": {
+                    "32400": {
+                        "description": "WebUI port. Suggested default: 32400",
+                        "host_default": 32400,
+                        "label": "WebUI port",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for Plex configuration. Eg: create a Share called plex-config for this purpose alone.",
+                        "label": "Config Storage"
+                    },
+                    "/data": {
+                        "description": "Choose a Share for Plex content(your media). Eg: create a Share called plex-data for this purpose alone. You can also assign other media Shares on the system after installation.",
+                        "label": "Data Storage"
+                    }
+                },
+                "environment": {
+                    "PLEX_UID": {
+                        "description": "Enter a valid UID to run Plex with. It must have full permissions to all Shares mapped in the previous step.",
+                        "label": "UID",
+                        "index": 1
+                    },
+                    "PLEX_GID": {
+                        "description": "Enter a valid GID to use along with the above UID. It(or the above UID) must have full permissions to all Shares mapped in the previous step.",
+                        "label": "GID",
+                        "index": 2
+                    }
+                }
+            }
+        },
+        "description": "Official Plex Media Server Docker Repo",
+        "more_info": "<h4>Adding more media to Plex.</h4><p>You can add more Shares(with media) to Plex from the settings wizard of this Rock-on. Then, from Plex WebUI, you can update and re-index your library.</p>",
```

Comply with Rockstor standard format ([rockstor/rockstor-core#1914](https://github.com/rockstor/rockstor-core/pull/1914)):
```suggestion
        "more_info": "<h4>Adding more media to Plex.</h4><p>You can add more shares (with media) to Plex from the settings wizard of this Rock-on. Then, from Plex WebUI, you can update and re-index your library.</p>",
```